scripts - Restore packages.bat

```diff
@@ -1,4 +0,0 @@
-for /f %%a in ('dir /b src\*.sln') do call tools\nuget\nuget.exe restore src\%%a -PackagesDirectory .\lib
```

Comment:
I never use msbuild for package restore. Not sure what you're referring to here. I just type "nuget restore" and it's done. It seems quicker than typing "scripts - Restore packages.bat"
Granted my way requires an extra step for the TestSolution.sln file.
What if we rename it to just restore.bat?